### PR TITLE
feat: support `placeholder` property

### PR DESCRIPTION
**Pull request by barmac**
This PR add support for `placeholder` property on `String` and `Text` fields.

Related to https://github.com/bpmn-io/bpmn-js-element-templates/issues/92

### CODE REVIEW

File: packages/zeebe-element-templates-json-schema/test/fixtures/placeholder.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      label: 'Input with placeholder',
+      type: 'Text',
+      binding: {
+        type: 'property',
+        name: 'prop'
+      },
+      placeholder: 'This field has a placeholder'
+    },
+    {
+      label: 'Input with placeholder',
```
Comment (philippfromme) [inline packages/zeebe-element-templates-json-schema/test/fixtures/placeholder.js:27]: Do I remember correctly that we assume a string/text type in these cases?

Comment (barmac) [reply packages/zeebe-element-templates-json-schema/test/fixtures/placeholder.js:27]: Yes, you do. This is why placeholder should be supported in such properties.

File: packages/zeebe-element-templates-json-schema/test/fixtures/placeholder-invalid-property.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+export const template = {
```
Comment (nikku) [inline packages/zeebe-element-templates-json-schema/test/fixtures/placeholder-invalid-property.js:1]: Question: We decided to only support it for C8?

**Review by nikku — APPROVED** [review]
Looks good to me. Just https://github.com/camunda/element-templates-json-schema/pull/144#discussion_r1618643590 to consider (parity between C7 and C8).

File: packages/zeebe-element-templates-json-schema/test/fixtures/placeholder-invalid-property.js (hunk 2 above)
Comment (barmac) [reply packages/zeebe-element-templates-json-schema/test/fixtures/placeholder-invalid-property.js:1]: We could implement it for both platforms, but that means greater effort in bpmn-js-element-templates project.

Comment (barmac) [reply packages/zeebe-element-templates-json-schema/test/fixtures/placeholder-invalid-property.js:1]: Let's have it in the core then.

**Pull request merged**